tests/reg_tests/test_euler_tutorial_wing.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import print_function
+import unittest
+import numpy
+from baseclasses import BaseRegTest
+import os
+import sys
+baseDir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(baseDir,'../../'))
+from python.pyADflow import ADFLOW
+import copy
+from baseclasses import AeroProblem
+# from reg_tests.commonUtils import standard_test, adflowDefOpts, defaultFuncList
+from commonUtils import *    
+from runpy import run_path
+
+
+class TestSolution(unittest.TestCase):
+    '''
+    Test 9: MDO tutorial -- Euler -- Solution Test
+    '''
+    N_PROCS = 4
+
+    def setUp(self):
+
+        self.ref_file = os.path.join(baseDir, 'ref/ref9.py')
+        ref = run_path(self.ref_file)['ref']
```

Comment:
Instead of using `run_path`, I think you can just import the file, using something like

`from ref.ref9 import ref`

Reply:
I was doing that originally, but i wanted to make sure training wrote to the same file so i thought it was better to use the file path when importing the reference

Reply:
Right that's a good point, okay.